components/radio/group.tsx

```diff
@@ -4,6 +4,7 @@ import classNames from 'classnames';
 import shallowEqual from 'shallowequal';
 import Radio from './radio';
 import { RadioGroupProps, RadioGroupState, RadioChangeEvent, RadioGroupButtonStyle } from './interface';
+import { polyfill } from 'react-lifecycles-compat';
```

Comment:
It's better to have absolute imports before relative, could you move it under `shallowEqual`